discovery-infra/tests/test_kube_api.py

```diff
@@ -264,3 +267,28 @@ def get_ignition_config_override(ca_bundle):
             }
     }
     return json.dumps(ignition_config_override)
+
+
+def collect_debug_info_from_cluster(cluster_deployment, agent_cluster_install):
+    cluster_name = cluster_deployment.ref.name
```

Comment:
nit: add a log msg "collecting debug info from cluster" so it will be easier to follow the failure flow from the logs

Reply:
right, added.